blocks/graph.py

```diff
@@ -244,12 +244,22 @@ def replace(self, replacements):
 
         return ComputationGraph(outputs_cur)
 
-    def get_theano_function(self, additional_updates=None):
-        """Create Theano function from the graph contained."""
+    def get_theano_function(self, additional_updates=None, **kwargs):
+        """Create Theano function from the graph contained.
```

Comment:
You have to prepend the docstring with 'r' here. Not sure if I really understand why, but this is what I was advised to do and it works.